### DISCUSSION TITLE
Synching favorites and Playlist with Subsonic

### DISCUSSION THREAD

**Discussion by frigi83** [music-assistant/support#2358] [Feature requests and ideas]
The usage of the music provider Subsonic is at the moment one way. On the otherway the provider Spotify has a synchronization in both ways. 

The idea is to sync favorites and Playlist in both ways. A change in MA will be visible on Subsonic  and the otherway. 

Others app that uses the Subsonic API, are able to do the synchronization. I think that should be possible to do that also in MA. But I am not a developer and I don't have any idea were to start. 

A solution like this allow to have the same Playlist in my home and on the way with the smartphone (done time offline).

**Comment by khers** [comment]
This PR begins to address your request: https://github.com/music-assistant/server/pull/1335

**Reply by frigi83** [reply]
Thank's @khers! I have tested the version [2.1.0 BETA 3](https://github.com/music-assistant/server/releases/tag/2.1.0b3). 

Works when adding a new provider, so I think that the issue #2359 can be closed.

I hope that synchronization in both directions, remains as the ultimate goal.

**Reply by khers** [reply]
The last piece of synchronization is favorites marked in MA sent back to the subsonic server. I will be working on that next.

**Comment by khers** [comment]
So it only took a year and a half, but this PR should complete the favorites syncing: https://github.com/music-assistant/server/pull/2663
